backend/adapters/angular2.ts

```diff
@@ -296,22 +296,16 @@ export class Angular2Adapter extends BaseAdapter {
     switch (name) {
       case 'NgFor':
         return this._normalizeNgFor(state);
-        break;
       case 'NgIf':
         return this._normalizeNgIf(state);
-        break;
-      case "NgClass":
+      case 'NgClass':
```

Comment:
Yea the "break;" was unreachable code; and double quotes had to be changed to single quotes.